miden-tx/src/testing/mock_chain/mod.rs

```diff
@@ -201,15 +205,27 @@ impl PendingObjects {
 ///
 /// # Examples
 ///
-/// ## Create mock objects
+/// ## Create mock objects and build a transaction context
 /// ```
-/// let mut mock_chain = MockChain::default();
+/// # use miden_tx::testing::{Auth, MockChain, TransactionContextBuilder};
+/// # use miden_objects::{assets::FungibleAsset, Felt, notes::NoteType};
+/// let mut mock_chain = MockChain::new();
 /// let faucet = mock_chain.add_new_faucet(Auth::BasicAuth, "USDT", 100_000);  // Create a USDT faucet
 /// let asset = faucet.mint(1000);  
-/// let note = mock_chain.add_p2id_note(asset, sender...);
 /// let sender = mock_chain.add_new_wallet(Auth::BasicAuth);  
-///
-/// mock_chain.build_tx_context(sender.id(), &[note.id()], &[]).build().execute()
+/// let target = mock_chain.add_new_wallet(Auth::BasicAuth);  
+/// let note = mock_chain
+///     .add_p2id_note(
+///         faucet.id(),
+///         target.id(),
+///         &[FungibleAsset::mock(10)],
+///         NoteType::Public,
+///       None,
+///     )
+///   .unwrap();
+/// mock_chain.seal_block(None);
+/// let tx_context = mock_chain.build_tx_context(sender.id(), &[note.id()], &[]).build();
+/// // tx_context.execute();
```

Comment:
If we can't make it a full example and have to comment this line, should we just add `no_run` (https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html#attributes)? That way it's compiled but not executed which seems fine here.

Reply:
Ah yes, that sounds good.